src/components/tracked-controls.js

```diff
@@ -128,16 +128,12 @@ module.exports.Component = registerComponent('tracked-controls', {
 
   handlePress: function (id, buttonState) {
     var buttonStates = this.buttonStates;
-    var evtName;
+    var evtName = buttonState.pressed ? 'down' : 'up';
     var previousButtonState = buttonStates[id] = buttonStates[id] || {};
-    if (buttonState.pressed === previousButtonState.pressed) { return; }
-    if (buttonState.pressed) {
-      evtName = 'down';
-    } else {
-      evtName = 'up';
-    }
+    if (buttonState.pressed === previousButtonState.pressed) { return false; }
     this.el.emit('button' + evtName, {id: id});
     previousButtonState.pressed = buttonState.pressed;
+    return true;
```

Comment:
it's an internal function, and handleTouch / handleValue follow the same convention without comments.  What would you like added?